lib/utils/sync/activity-hooks.js

```diff
@@ -0,0 +1,47 @@
+import { debounce, isFunction, noop, startsWith } from 'lodash';
+import Debug from 'debug';
+
+const debug = Debug('sync:activityHooks');
+
+export const onSyncActive = debounce(
+  (data, onActive) => {
+    debug('Sync active...');
+
+    if (isFunction(onActive)) {
+      onActive();
+    }
+  },
+  500,
```

Comment:
Maybe create a constant for these two `500` values?

Reply:
Good idea!